modules/stanford_events_importer/tests/src/Unit/StanfordEventsImporterTest.php

```diff
@@ -139,22 +151,21 @@ public function testParseXML() {
 
   /**
    * Make sure exception passes back false.
+   *
    * @return [type] [description]
    */
   public function testFetchXMLException() {
-    // Create a mock and queue two responses.
-    $mock = new MockHandler([
```

Comment:
You don't like this?

Reply:
:shrug: mostly i didn't like the `$client->get` method use. so i changed that up.